### DISCUSSION TITLE
FanControl Unable to pair RPM monitor to fans in Arctic 360 mm AIO

### DISCUSSION THREAD

**Discussion by pedrorribeiro** [Rem0o/FanControl.Releases#852] [Q&A]
Hi, just started using FanControl, loving it so far, I found it through JayzTwoCentz's video, so I used that as a guideline to get the software set up, then I did some testing of my own by setting up custom graphs and monitoring the CPU and GPU temps while using Cinebench and Elden Ring on max quality. Everything is working great, I found my top and rear fans, and the GPU fans showed up from the start. I know the three fans that remain are my AIO fans, but I can't really do anything with them on FanControl, because there isn't an RPM monitor, how can I fix this issue?

**Comment by Rem0o** [comment]
Plug them onto motherboard headers?

**Reply by pedrorribeiro** [reply]
Hi, they are plugges in to motherboard headers.

**Reply by Rem0o** [reply]
Your issue is not clear to me, what do you mean there isn't a RPM monitor? How are they plugged to your mobo? Any splitter? Screenshots do wonders explaining things.

**Reply by OutsideOctaves** [reply]
I've got the Arctic Freezer II (a 240 I believe) and I have a similar issue I think.  Mine is plugged into the CPU_Fan header, and if I leave the switch on the controller off it works, but as soon as I enable it the fans stop.  I have it hooked to the Auto (beta) function, but even with the flat example set to 100, 50, or et cetera percents it still doesn't work for some odd reason.  This particular AIO uses a single connection to control pump, the fans, and the little fan on the pump that cools the VRM.  I just wonder what's going on here myself.  I'm fine for letting the MoBo do  it's thing for now, but am curious if there's a fix for this with a settings tweak somewhere.

**Comment by JuJu30420** [comment]
I have the same issue but with all my fans, except for the GPU. As soon as I enable the fan (switch ''on'' for CPU fan, or any other fan header) the fan just stop. It start again when I switch off the fan header. It start happening since couple days. Some say it's a Windows update and to fix it is to re-enabling .NET Framework 3.5, but it's already enabled.. I tried to reinstalled it manually, but same issue. If this can't work, will need to use another software.
